app/src/components/job/hr/CreateJob.jsx

```diff
@@ -26,6 +29,9 @@ export default function HRCreateJob() {
       setErrors(errorList);
     } else {
       setErrors([]);
+      selectedSkills.map(async (skillId) => {
```

Comment:
The use of map should be to return an array. Use a for loop if you want to do actions or else the event loop will bug out